File: p3232/cpp/main.cpp
class Solution
{
public:
    bool canAliceWin(vector<int> &nums)
    {
        int single=0, ddouble = 0;
        for (int i = 0; i < nums.size(); i++)
        {
            if (nums[i] <= 9)
            {
                single +=nums[i];
            }else{
                ddouble +=nums[i];
            }
            
        }
        if (single == ddouble)
        {
            return false;
        }

        return true;
    }
};

File: p3232/rust/src/main.rs
struct Solution;
impl Solution {
    pub fn can_alice_win(nums: Vec<i32>) -> bool {
        let mut sum_single = 0;
        let mut sum_double = 0;
        for n in nums {
            if n <= 9 {
                sum_single += n;
            }else {
                sum_double += n;
            }
        }
        if sum_double == sum_single {
            return false;
        }
        true // alice win
    }
}
fn main() {
    println!("Hello, world!");
}
